test/Microsoft.AspNetCore.Mvc.Analyzers.Test/Helpers/CodeFixVerifier.Helper.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Simplification;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace TestHelper
+{
+    /// <summary>
+    /// Diagnostic Producer class with extra methods dealing with applying codefixes
+    /// All methods are static
+    /// </summary>
+    public abstract partial class CodeFixVerifier : DiagnosticVerifier
```

Comment:
FYI - Most of this code is from the Roslyn analyzer template. I cleaned up some of it to make it compliant with xUnit and some `var` usage, but left most of it as is.